Dockerfile

```diff
@@ -68,6 +68,7 @@ RUN	apt update && apt install -yqq \
 COPY --from=catalyst-build	/opt/bin/		/usr/local/bin/
 COPY --from=node-build		/app/go-tools/w3	/opt/local/lib/livepeer-w3
 RUN	ln -s /opt/local/lib/livepeer-w3/livepeer-w3.js /usr/local/bin/livepeer-w3
+RUN	npm install -g ipfs-car
```

Comment:
both these `RUN` statements should be merged, to have 1 less layer!

Reply:
Right, updated.